### PR TITLE
Don't block buttons in views

### PR DESCRIPTION
**Pull request by Mik-chan**
Just one line fix

### CODE REVIEW

File: DragDropUI/DD+View.swift [hunk 1]
```diff
@@ -33,10 +33,11 @@ public extension DDProtocol where Self: UIView {
 
     func registerGesture() {
         let panGesture = UIPanGestureRecognizer()
+        panGesture.cancelsTouchesInView = false
         panGesture.handler = { gesture in
             self.handlePan(panGesture: gesture as! UIPanGestureRecognizer)
         }
-
+
```
Comment (abdullahselek) [inline DragDropUI/DD+View.swift]: Please don't leave 4 empty spaces in empty lines.

File: DragDropUI/DD+View.swift [hunk 2]
```diff
@@ -36,10 +36,11 @@ public extension DDProtocol where Self: UIView {
...
```
Comment (abdullahselek) [inline DragDropUI/DD+View.swift]: Please don't leave 4 empty spaces in empty lines.

**Review by abdullahselek — COMMENTED** [review]
Thanks for the small fix, please do some minor changes.

File: DragDropUI/DD+View.swift (hunk 2 above)
Comment (Mik-chan) [reply DragDropUI/DD+View.swift]: Ok, done!

**Review by abdullahselek — APPROVED** [review]

**Comment by abdullahselek** [comment]
Thanks for your contribution 👍

**Pull request merged**